@aandvalenzuela , the test results report "Clang Warnings to fix". Could you please have a look ?
```
RecoTauTag/HLTProducers/src/L2TauPixelTrackMatch.cc:82:14: warning: unused variable 'n0' [-Wunused-variable]
```

--- a/RecoTauTag/HLTProducers/src/L2TauPixelTrackMatch.cc
+++ b/RecoTauTag/HLTProducers/src/L2TauPixelTrackMatch.cc
@@ -79,8 +79,6 @@ void L2TauPixelTrackMatch::produce(edm::StreamID, edm::Event& ev, const edm::Eve
       if (jet->pt() < m_jetMinPt || std::abs(jet->eta()) > m_jetMaxEta)
         continue;
 
-      size_t n0 = new_tau_jets->size();
-
       for (vector<TinyTrack>::const_iterator itrk = good_tracks.begin(); itrk != good_tracks.end(); ++itrk) {
         math::XYZTLorentzVector new_jet_dir = Jet::physicsP4(itrk->vtx, *jet, itrk->vtx);
         float dphi = reco::deltaPhi(new_jet_dir.phi(), itrk->phi);

